### PR TITLE
Extend table making for DPG

### PR DESCRIPTION
**Pull request by njacazio**
- Add table for the MC
- Add table for the collision

### CODE REVIEW

**Comment by njacazio** [comment]
@mfaggin @mario-krueger I propose this extension to access also A) event info and B) MC

**Review by mario-krueger — APPROVED** [review]

File: DPG/Tasks/qaEventTrack.cxx [hunk 1]
```diff
@@ -235,9 +277,53 @@ struct qaEventTrackLite {
   Filter tpcNcrossedRows = o2::aod::dpgtrack::tpcNClsCrossedRows > (int16_t)nCrossedRowsTpcMin;
   Filter tpcNcrossedRowsOverFindableClusters = o2::aod::dpgtrack::tpcCrossedRowsOverFindableCls > nCrossedRowsTpcOverFindableClustersTpcMin;
 
-  void process(o2::soa::Filtered<aod::DPGTracks> const& tracks)
+  // Process data
+  void processDataLite(o2::soa::Filtered<aod::DPGTracks> const& tracks, aod::DPGCollisions const&)
+  {
+    for (const auto& track : tracks) {
+      histos.fill(HIST("Tracks/VertexPositionZ"), track.dpgCollision().posZ());
```
[the remark this answers is not in the document]
Comment (njacazio) [reply DPG/Tasks/qaEventTrack.cxx:284]: all skimmed tracks should have a collision

File: DPG/Tasks/qaEventTrack.cxx [hunk 2]
```diff
@@ -296,6 +383,10 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
 //**************************************************************************************************
 void qaEventTrack::init(InitContext const&)
 {
+  if (doprocessTableData == true && doprocessTableMC == true) {
```
[the remark this answers is not in the document]
Comment (njacazio) [reply DPG/Tasks/qaEventTrack.cxx:386]: `doprocessTableData` is created in the `PROCESS_SWITCH(processTableData`

[the remark this answers is not in the document]
Comment (mfaggin) [reply DPG/Tasks/qaEventTrack.cxx:386]: ah yes they are flags of the `PROCESS_SWITCH`, my bad

[the remark this answers is not in the document]
Comment (njacazio) [reply DPG/Tasks/qaEventTrack.cxx:386]: indeed the code is a bit messy, we should refactor it at some point

File: DPG/Tasks/qaEventTrack.cxx (hunk 1 above)
[the remark this answers is not in the document]
Comment (mfaggin) [reply DPG/Tasks/qaEventTrack.cxx:284]: that's because we have already the grouping in `processTableData` and `processTableMC`, my bad again

**Pull request merged**